src/IndexedRecord.php

```diff
@@ -2,11 +2,18 @@
 namespace Swis\LaravelFulltext;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Config;
 
 class IndexedRecord extends Model
 {
     protected $table = 'laravel_fulltext';
 
+    public function __construct()
+    {
+        parent::__construct();
+        $this->connection = Config::get('laravel-fulltext.db_connection', 'mysql');
```

Comment:
Can we look at the `config('database.default')` value here instead of the hard coded mysql?

Reply:
Thanks for your feedback, i fixed all of it